monitoring/opentelemetry-reactive/src/test/java/io/quarkus/ts/opentelemetry/reactive/OpenTelemetryGrpcIT.java

```diff
@@ -0,0 +1,65 @@
+package io.quarkus.ts.opentelemetry.reactive;
+
+import static io.restassured.RestAssured.given;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.concurrent.TimeUnit;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.JaegerService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.JaegerContainer;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class OpenTelemetryGrpcIT {
+
+    @JaegerContainer(useOtlpCollector = true, image = "quay.io/jaegertracing/all-in-one:1.41.0")
```

Comment:
I guess I should drop `quay.io/jaegertracing/all-in-one:1.41.0` occurrences and update the default in framework.

Waiting till the review is done and CI run finished.

Reply:
My PR worked with both versions, we can merge. I will handle version bump in the FW by the end of the week.